e2e-tests/portal/cypress/fixtures/mockFacilities.js

```diff
@@ -0,0 +1,36 @@
+const MOCK_BOND = {
+  type: 'Bond',
+  bondIssuer: 'issuer',
+  bondType: 'bond type',
+  facilityStage: 'unissued',
+  ukefGuaranteeInMonths: '24',
+  name: '1234',
+  bondBeneficiary: 'test',
+  value: '123',
+  currencySameAsSupplyContractCurrency: 'true',
+  riskMarginFee: '1',
+  coveredPercentage: '2',
+  feeType: 'test',
+  feeFrequency: 'Quarterly',
+  dayCountBasis: 'test',
```

Comment:
Are these just copy pasted from somewhere else? Wonder if we should put some legit values in here like 360 and feeType as `In arrear'